pkg/config/load.go

```
@@ -0,0 +1,352 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	clog "github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported as "clog" but must be "log" according to config

(importas)